app/controllers/api/v1/emotions_controller.rb

```diff
@@ -73,4 +77,9 @@ def emotion_params
   def time_period
     @time_period ||= TimePeriod.find_or_create_time_period
   end
+
+  def list_shoutouts_to(user)
+    shoutout_ids = ShoutoutRecipient.where(user_id: user.id).pluck(:shoutout_id)
```

Comment:
It's always preferable to incorporate model relationships than to start with base model queries. Is there a reason why `user.shoutout_recipients.pluck(:shoutout_id)` can't be used?